src/App.css

```diff
@@ -36,3 +36,47 @@
     transform: rotate(360deg);
   }
 }
+
+.button {
+  background-color: #4CAF50;
+  border: none;
+  color: white;
+  padding: 12px 24px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  border-radius: 8px;
+  transition: background-color 0.3s ease, transform 0.2s ease;
+  cursor: pointer;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.2);
+  margin-bottom: 8px;
+  min-width: 300px;
+  width: 100%;
+}
+
+.button:hover {
+  background-color: #45a049;
```

Comment:
The button ends up on top of the textbox when hovering and not otherwise. Can that be avoided?

Reply:
Could you supply a screenshot please? That error does not seem to show up for me.

Reply:
Here:
<img width="893" alt="image" src="https://github.com/user-attachments/assets/58d1d42e-6c4f-4575-8673-8cf877b36fae">

Reply:
I just removed the effect now since it isn't necessary